patroni/dcs/__init__.py

```diff
@@ -257,7 +257,27 @@ def tags(self) -> Dict[str, Any]:
 
     @property
     def nofailover(self) -> bool:
-        return self.tags.get('nofailover', False)
+        from_tags = self.tags.get('nofailover', None)
+        if from_tags is not None:
+            # Value of `nofailover` takes precedence over `failover_priority`
+            return bool(from_tags)
+        failover_priority = self.tags.get('failover_priority', 1)
+        return int(failover_priority) <= 0
+
+    @property
+    def failover_priority(self) -> int:
+        from_tags = self.tags.get('nofailover', None)
+        fp = self.tags.get('failover_priority', 1)
+        fp = int(fp) if fp is not None else 1
```

Comment:
The `int()` could raise an exception.

```suggestion
        fp = parse_int(self.tags.get('failover_priority', 1)) or 1
```

Reply:
what if `failover_priority` is 0? Then won't `fp` evaluate to `1` because `0 or 1` would evaluate to 1? What about:
```
fp = parse_int(self.tags.get('failover_priority', 1))
fp = 1 if fp is None else fp
```